DotnetFoundation/DotnetFoundation.Api/appsettings.json

```diff
@@ -9,9 +9,14 @@
     "DBConnection": "server=foundation-mariadb;Port=3306;database=DotnetFoundation;Uid=root;pwd=Change123"
   },
   "Jwt": {
-    "Key": "My-Super-Secret-Key!@#$%",
+    "Key": "ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAACAQC2CoxBJo5TR2O2agbJ3rvoQAC1CGNOfcGWlyxzUcIfoXBLoDjimzfb/nIfvku3m4ZTzsx88+mJw4gtNo3xx3T4nVszGTOEeTYZod9WlekilKtWMCCYmUrPbdCOVsIgiD4om3yDvoIarVd28PymS7Ub81rYhMFsm+Pmq3Ms5l/VXkSdyh8D60Z6KjDQh39EVs6+qzY3PJHNnZgLx9ivz40mCmOkYd2e58U/z0kX20APr/+zJZtZJm15uFCm4C3aRzQ78lQ7cZtt3pE3Bsth1wS4/utwYWm930YCNMqZeYBNn0n6qfTOkS/wDFJpKNgC3+7v4THoqMuzAO/w7hKDsjSP4vJ2bm1tnoB0yEjkK2SMT85wNxWl7JkIRWW1FqxdWMQD5c1r3g5TvAzfPdJ/rw5zZNIUyq2r/7lHJrO1UGEsmSddOvBEk4uQ4J905xPgxT8w3Grbz0QjHvZh+AoYRu988b5Nc/3NOVf8EWqnOf55vRW9lxU51r40dpcQPkBik3Q0F1r1VSpPHvr0H0S+aSa+dgim4fnzA/mEcD/x2uUMQP0RbEHnsU/dR7fMtSqUVEDKD9nlvg/TW2lWme/sW2L/Dg1//bUODrdgtimEjBbWB+Ce9HG/+Os5QaFqp9Fgr9ADGlMOFlLCja8CkR+e0DEcJ+9kf7pl0dsy2XzfXUUr6Q== work@Windows",
```

Comment:
Create an .env file and use it to replace the text in the appSettings.json file in the Dockerfile.

Reply:
resolved